src/Symfony/Component/HttpClient/Response/AsyncContext.php

```diff
@@ -25,12 +25,15 @@
 final class AsyncContext
 {
     private $passthru;
-    private $client;
-    private $response;
-    private $info = [];
+    private HttpClientInterface $client;
+    private ResponseInterface $response;
+    private array $info = [];
     private $content;
-    private $offset;
+    private int $offset;
 
+    /**
+     * @param resource|null $content
+     */
     public function __construct(?callable &$passthru, HttpClientInterface $client, ResponseInterface &$response, array &$info, $content, int $offset)
```

Comment:
unrelated to your pr again. but i don't get why this is by ref again `ResponseInterface &$response`

Reply:
that how `AsyncContext` is able to change the inner response of an `AsyncResponse`